lib/modules/datasource/git-refs/base.ts

```diff
@@ -22,7 +22,7 @@ export abstract class GitDatasource extends Datasource {
   }
 
   @cache({
-    namespace: `datasource-${GitDatasource.id}`,
+    namespace: 'datasource-git',
```

Comment:
Sorry for that, `GitDatasource.id` can't be marked as const because of inheritance

Reply:
can you extract the id to a separate const and reuse instead?